lua/lsp/providers/python.lua

```diff
@@ -0,0 +1,11 @@
+local opts = {
```

Comment:
I used to be able to place my null-ls formatter configurations in my ftplugin folder in `~/.config/lvim`  but that is no longer possible.  I have to place `lvim.lang.python.formatters` directly in my config.lua

Reply:
I didn't really do anything to the autocmd logic except deleting the folder `ftplugin` only because I didn't actually check if there are non-lsp related files. So can you try creating an empty `ftplugin` folder and see if that was reason?